### ISSUE TITLE
add AES-256 media stream encryption control to peerconnection

### ISSUE THREAD

**Issue by bdrtc** [w3c/webrtc-extensions#113]
There is no standard WebRTC API for managing the DTLS-SRTP cipher priority order  currently, and the media stream are encrypted using AES-128 by default, for enhanced security, we suggest add api to open the AES-256 encryption as option.

**Comment by bdrtc** [comment]
there was some discusstion about this , but no result on export js api , [](https://groups.google.com/g/discuss-webrtc/c/fz3khhADJNc) , and  some webrtc base rtc platform use AES-256 as addcanced feature for enhanced security.

**Comment by lidp888** [comment]
agree, we implement this by change native code as an add-on feature of our product, but for web, there was no such api.

**Comment by changzhn** [comment]
I recently received such requirement from my customer for enhance security , I can modify the native client to achive this ,but can do nothing for web

**Comment by bdrtc** [comment]
the [article](https://webrtchacks.com/meet-vs-duo-2-faces-of-googles-webrtc/) hack the google-meet&duo, meet uses the more modern and efficient AEAD_AES_256_GCM cipher for encryption.
